tests/cogs/insights/test_cog.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+"""
+Testing KoalaBot Insights Cog
+"""
+# Futures
+
+# Built-in/Generic Imports
+
+# Libs
+import discord
+import discord.ext.test as dpytest
+import mock
+import pytest
+import pytest_asyncio
+
+# Own modules
+import koalabot
+from koala.cogs.insights import cog
+
+# Constants
+
+# Variables
+
+
+@pytest.mark.asyncio
+async def test_setup(bot):
+    with mock.patch.object(discord.ext.commands.bot.Bot, 'add_cog') as mock1:
+        await cog.setup(bot)
+    mock1.assert_called()
+
+
+@pytest_asyncio.fixture
+async def insights_cog(bot: discord.ext.commands.Bot):
+    """ setup any state specific to the execution of the given module."""
+    insights_cog = cog.Insights(bot)
+    await bot.add_cog(insights_cog)
+    await dpytest.empty_queue()
+    dpytest.configure(bot)
+    return insights_cog
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("members,guilds", [(0, 0), (10, 10), (100, 100), (1000, 1000)])
+async def test_insights(bot, insights_cog, members, guilds):
+    for x in range(members):
+        await dpytest.member_join(0, name=f"TestUser{x}", discrim={x})
+
+    for x in range(guilds):
+        guild = dpytest.backend.make_guild(name=f"FakeGuild{x}")
+        dpytest.get_config().guilds.append(guild)
+        await dpytest.member_join(guild, dpytest.get_config().client.user)
+    
+    total_guilds = 1 + guilds
+    total_members = 2 + guilds + members
+    
+    await dpytest.message(koalabot.COMMAND_PREFIX + "insights")
+
+    expected_message = (f"Insights:\nThis bot is in a total of {total_guilds} servers."
+                        f"\nThere are a total of {total_members} members across these servers.")
```

Comment:
Should follow consistent approach for multiple lines (+ instead of brackets)
```suggestion
    expected_message = f"Insights:\nThis bot is in a total of {total_guilds} servers." +
                        f"\nThere are a total of {total_members} members across these servers."
```